index.md

```diff
@@ -1,57 +1,198 @@
+				<img id="switch-logo" class="cross-platform-logo" loading="lazy">
+			</div>
+		</div>
+		<div class="col-12 col-md-6">
+			<h3 class="fw-semibold mb-4 text-body-emphasis">Cross-Platform</h3>
+			<p class="lead">
+				Build your game for multiple platforms.  MonoGame currently supports:
+				<div class="row">
+					<div class="col-12 col-md-4">
+						<h5>Desktop</h5>
+						<ul class="list-unstyled">
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Windows</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> MacOS</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Linux</li>
+						</ul>
+					</div>
+					<div class="col-12 col-md-4">
+						<h5>Mobile</h5>
+						<ul class="list-unstyled">
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Android</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> iOS</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> iPadOS</li>
+						</ul>
+					</div>	
+					<div class="col-12 col-md-4">
+						<h5>Console <i class="color-monogame fs-4">*</i</h5>			
+						<ul class="list-unstyled">
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Playstation 4</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Playstation 5</li>
+							<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> Xbox One</li>
+							<li class="mb-3 lead text-nowrap"><i class="bi bi-check-lg color-monogame pe-2"></i> Nintendo Switch</li>
+						</ul>
+					</div>								
+				</div>
+			</p>
+		</div>
+	</div>
+	<div class="row">
+	 <p class="text-secondary">
+	 	<i class="color-monogame fs-4">*</i> Console exporting requires access to the SDK for that console.  After completing the sign up processes for the console, your representative will be able to walk you through the steps of gaining accesses to the tools for exporting MonoGame to that console.
+	</p>
+	</div>
+</section>
+
+<section class="container-xxl my-5">
+	<div class="row">
+		<div class="col-12 col-md-6">
+			<h3 class="fw-semibold mb-4 text-body-emphasis">Open-Source</h3>
+			<p class="lead">
+				All the code is available to you ensuring you'll have the ability to make changes when you need to or even port to whole new platforms. This means MonoGame is free to use, forever.
+		   </p>
+		   <ul class="list-unstyled">
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> No licensing cost!</li>
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> No royalties!</li>
+			<li class="mb-3 lead"><i class="bi bi-check-lg color-monogame pe-2"></i> No runtime fee!</li>
+		   </ul>
+		</div>
+		<div class="col-12 col-md-6 d-flex flex-row justify-content-center">
+			<img id="github-logo" class="featurette-image object-fit-contain" loading="lazy">

```

Comment:
My first impression is some of these icons around these sections seem overly large:

![image](https://github.com/MonoGame/monogame.github.io/assets/103522/56612432-b4f2-4b1e-b250-2a7a501de5c9)

Reply:
Adjusted the size of the GitHub logo image here.  Also updated the managed-code image to fix the tilt perspective on the game window overlaying the code window (they were misaligned).  

I didn't feel that the managed-code image was too large, but I'll defer to your recommendation for it